src/_11ty/filters.js

```diff
@@ -1,49 +1,49 @@
-const date = require('nunjucks-date');
+export { default as date } from 'nunjucks-date';
 
-const encodeForUrl = string => encodeURIComponent(string)
+export function encodeForUrl(string) {
+  return encodeURIComponent(string)
+}
 
-function readableDate(dateObj) {
+export function readableDate(dateObj) {
   return new Date(dateObj).toDateString()
 }
 
-function upcomingEvents(events) {
+export function upcomingEvents(events) {
   return events.filter(event => {
     return new Date(event.data.event.startAt.utc) > new Date()
   }).reverse()
 }
 
-module.exports = {
-  calendarDescription: description => {
-    const text = `${description} See https://11tymeetup.dev/ for the full details!`
-    return encodeForUrl(text)
-  },
-  date,
-  encodeForUrl,
-  readableDate,
-  readableDateTime: dateObj => {
-    // "Jun 25, 2021, 12:00:00 PM CDT"  <--wanted time zone
-    return new Date(dateObj).toLocaleString([],{
-      day: '2-digit',
-      month: "short",
-      year: "numeric",
-      hour: "numeric",
-      minute: "numeric",
-      timeZoneName: "short",
-    })
-  },
-  nextEvent: events => {
-    const filteredEvents = upcomingEvents(events)
-    return filteredEvents.length > 0 ? filteredEvents[0].data.event : null
-  },
-  pastEvents: events => {
-    return events.filter(event => {
-      return new Date(event.data.event.startAt.utc) < new Date()
-    })
-  },
-  upcomingEvents,
-  excerpt: (post, limit="300") => {
-    const content = post.replace(/(<([^>]+)>)/gi, "");
-    content.length > limit ? content.substr(0, content.lastIndexOf(" ", limit)) + "..." : content;
-    return content;
-  }
+export function calendarDescription(description) {
```

Comment:
This diff looks worse than it is, you can "Hide whitespace" in the GitHub UI (cog in the top right) to get a more sensible diff here.